ci-operator/step-registry/gcp/provision/private-dns-zone-in-service-project/gcp-provision-private-dns-zone-in-service-project-ref.metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+	"path": "gcp/provision/private-dns-zone-in-service-project/gcp-provision-private-dns-zone-in-service-project-ref.yaml",
```

Comment:
How about introduce `gcp/provision/private-dns-zone/in-service-project/` and `gcp/provision/private-dns-zone/in-host-project/`